understanding/focus-not-obscured-minimum.html

```diff
@@ -380,90 +275,62 @@ <h4>User openable, persistent disclosures</h4>
                </section>
                <section id="dialogs">
                   
-                  <h3>Modal dialogs</h3>
+                  <h3>モーダルダイアログ</h3>
                   
                   
-                  <p>A properly constructed modal dialog will always pass this SC. Even if it appears directly
-                     on top of an item with focus, the dialog takes focus on appearance, and thus the item
-                     receiving focus -- the dialog or one of its components -- is visible. A properly constructed
-                     modal maintains that focus and prevents interaction outside the modal until it is
-                     dismissed.
+                  <p>適切に構成されたモーダルダイアログは、常にこの達成基準に合格する。たとえフォーカスを持つ項目の上に直接表示されたとしても、ダイアログは外観上フォーカスを受け取り、したがってフォーカスを受け取る項目 (ダイアログ又はそのコンポーネントの一つ) が表示される。適切に構成されたモーダルは、そのフォーカスを維持し、モーダルが閉じられるまでモーダル外のインタラクションを防ぐ。
                   </p>
                   
-                  <p>A dialog-like overlay that does not take focus on appearance and does not either constrain
-                     interaction to the overlay or dismiss itself on loss of focus (thus allowing focus
-                     to exit into the content behind it) will be at risk of failing this SC, where it is
-                     positioned such that it can obscure other focusable items.
+                  <p>外観上でフォーカスを受け取らず、かつオーバーレイに対してインタラクションを制限しない、又はフォーカスを失ったときに自身を閉じない (したがって、フォーカスが背後のコンテンツに抜けるようにする) ダイアログのようなオーバーレイは、他のフォーカス可能な項目を隠すことができるように配置されており、この達成基準で失敗する危険性がある。
                   </p>
                   
                </section>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   
-                  <li>Sighted users who rely on a keyboard interface to operate the page will be able to
-                     see the component which gets keyboard focus. Such users include those who rely on
-                     a keyboard or on devices which use the keyboard interface, including speech input,
-                     sip-and-puff software, onscreen keyboards, scanning software, and a variety of assistive
-                     technologies and alternate keyboards.
+                  <li>キーボードインタフェースに依存してページを操作している目の見える利用者は、キーボードフォーカスを取得するコンポーネントを見ることができる。そのような利用者には、キーボード又は、音声入力、呼気・吸気ソフトウェア、オンスクリーンキーボード、スキャンソフトウェア、並びにさまざまな支援技術及び代替キーボードなどのキーボードインタフェースに依存している利用者が含まれる。
```

Comment:
"devices" の訳が落ちている気がするのと、語順を原文に近い形にした方が読みやすいように思います。こんな感じてどうでしょうか。

```suggestion
                  <li>キーボードインタフェースに依存してページを操作している目の見える利用者は、キーボードフォーカスを取得するコンポーネントを見ることができる。そのような利用者は、キーボード又はキーボードインタフェースを利用するデバイスに依存している場合があり、音声入力、呼気・吸気ソフトウェア、オンスクリーンキーボード、スキャンソフトウェア、並びにさまざまな支援技術及び代替キーボードなどが含まれる。
```

Reply:
かっこ書きするのはどうでしょう。

```suggestion
                  <li>キーボードインタフェースに依存してページを操作している目の見える利用者は、キーボードフォーカスを取得するコンポーネントを見ることができる。そのような利用者には、キーボード又はキーボードインタフェースを利用するデバイス (音声入力、呼気・吸気ソフトウェア、オンスクリーンキーボード、スキャンソフトウェア、並びにさまざまな支援技術及び代替キーボードなど) に依存するような利用者が含まれる。
```